README.md

```diff
@@ -94,6 +94,41 @@ The component which orchestrates other components: a "glue" component. Provides
 a way to start the bridge. This is the component most examples use. Has
 dependencies on most of the components listed above.
 
+## `Logging`
+This component exposes access to the bridges log reporter. To use, you should
+install the optional packages `winston`, `winston-daily-rotate-file` and
+`chalk` to get nice formatted log lines, otherwise it will default to the JS
+console. To use the component, use `Logging.Configure(configObject)` to setup
+the logger, which takes the following options:
+```
+    # A level to set the console reporter to.
+    console: "error|warn|info|debug|off"
+
+    # Format to append to log files.
+    fileDatePattern: "YYYY-MM-DD",
```

Comment:
the others don't have a comma, and they probably should. Can we get away with putting the surrounding curly braces around the whole snippet and syntax highlight it for javascript? This is purely aesthetic and nothing else